invisible_cities/cities/components.py

```diff
@@ -356,6 +358,8 @@ def mcsensors_from_file(paths     : List[str],
                  Name of detector database to be used
     run_number : int
                  Run number for database
+    rate       : float
+                 Rate value in Hertz to generate timestamps
```

Comment:
So the number (if we use the units correctly) won't be in Hertz. It can be a bit confusing at first but units in this sense convert everything into a base unit. In the case of time we don't define seconds as the base unit (`=1.0`) so Hertz isn't the base unit for rate/frequency either.

The line in the configuration `0.5 * hertz` actually multiplies 0.5 by `1e-9` since the time base unit is `ns`.

I think the least confusing comment might be something like "rate in base unit (ns^-1) but maybe @mmkekic or @jmalbos have an opinion.

Reply:
Right, by reading `system_of_units.py` I have understood the concept about units. Thank you for the explanation

I can wait for other comments and then change the docstring.